x/slashing/app_test.go

```diff
@@ -55,7 +55,7 @@ func getEndBlocker(keeper stake.Keeper) sdk.EndBlocker {
 func getInitChainer(mapp *mock.App, keeper stake.Keeper) sdk.InitChainer {
 	return func(ctx sdk.Context, req abci.RequestInitChain) abci.ResponseInitChain {
 		mapp.InitChainer(ctx, req)
-		stake.InitGenesis(ctx, keeper, stake.DefaultGenesisState())
+		keeper.InitGenesis(ctx, stake.DefaultGenesisState())
```

Comment:
Last I recall, we decided we wanted this *not* to be a function on the keeper for the same reason `NewHandler` isn't a function on the keeper - ref https://github.com/cosmos/cosmos-sdk/issues/689.

Reply:
agree / moved back to root of stake